enterprise/models.py

```diff
@@ -155,6 +155,14 @@ class Meta:
         )
     )
 
+    replace_sensitive_sso_username = models.BooleanField(
+        default=False,
+        help_text=_(
+            "Specifies whether to replace the display of potentially sensitive SSO usernames "
+            "with a more generic name, i.e. EnterpriseLearner."
```

Comment:
@UmanShahzad Nit: It's not that exact string, right (`EnterpriseLearner`)? Maybe say `<Enterprise>Learner`, or just replace `i.e.` with `e.g.`? :)

Reply:
It was just meant as an example, yes, so `.e.g.` sounds fine to use.

Reply:
:+1: